webapp/src/components/modals/create_issue/create_issue_form.tsx

```diff
@@ -136,7 +136,8 @@ export default class CreateIssueForm extends React.PureComponent<Props, State> {
         });
 
         let fields = {
-            ...this.state.fields,
+            summary: this.state.fields.summary,
+            description: this.state.fields.description,
```

Comment:
Can we get rid of the `filterInvalidFields` function in this file, since we aren't checking the arbitrary fields anymore? It may be used for something later such as https://github.com/mattermost/mattermost-plugin-jira/issues/855

Reply:
@mickmister removed the function